joern-cli/frontends/rubysrc2cpg/src/test/scala/io/joern/rubysrc2cpg/querying/RubyMethodFullNameTests.scala

```diff
@@ -0,0 +1,51 @@
+package io.joern.rubysrc2cpg.querying
+
+import io.joern.rubysrc2cpg.testfixtures.RubyCode2CpgFixture
+import io.shiftleft.semanticcpg.language._
+
+class RubyMethodFullNameTests extends RubyCode2CpgFixture {
```

Comment:
Can we add more test cases? Ex - 
- class defined in a package, 
- For the call Node is there a method with the same name and methodFullName

Reply:
Yes sure will add.

Reply:
done